security/locale/cs/LC_MESSAGES/django.po

```diff
@@ -18,203 +18,211 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
 
-#: cores.py:32 models.py:129 models.py:160
+#: cores.py:30 models.py:168 models.py:199
```

Comment:
ouch, we should use `--no-location` when generating this file.

Reply:
@zzuzzy can you please do it with that option?

Reply:
Ok, location is now removed.